custom_components/tuya_local/devices/poolsana_heatpump.yaml

```diff
@@ -0,0 +1,85 @@
+name: Pool heat pump
+primary_entity:
+  products:
+  - id: abb8kaxcphfzkwsg
+    name: PoolSana InverPower Next
+  entity: climate
+  translation_key: pool_heatpump
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cooling
+              value: cool
+            - dps_val: Heating
+              value: heat
+            - dps_val: Auto
+              value: auto
```

Comment:
This more likely corresponds to HA's `heat_cool` mode than `auto` based on the descriptions at https://developers.home-assistant.io/docs/core/entity/climate/#hvac-modes

Reply:
Yeah, you're right, it's no AI behind :-) Corrected